config/routes.yaml

```diff
@@ -2,4 +2,16 @@
 #    path: /
 #    controller: App\Controller\DefaultController::index
 app_logout:
-  path: /logout
\ No newline at end of file
+  path: /logout
+
+ConfirmRegistration:
```

Comment:
It seems to be snake_case everywhere, I think we should stay coherent with ourselves on these things.

Reply:
Done